Products/ZenHub/zenhub.py

```diff
@@ -414,7 +414,7 @@ def __init__(self, parser):
         self.parser = parser
 
 
-def report_reactor_delayed_calls(hub):
+def report_reactor_delayed_calls(monitor, component):
```

Comment:
`component` can be just called `name` now.

Reply:
fixed